### ISSUE TITLE
Error after flutter upgrade

### ISSUE THREAD

**Issue by rlee1990** [GetStream/stream-video-flutter#813]
I get the below message after upgrading flutter:
```dart
Package stream_video_push_notification:android references stream_video_push_notification:android as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_video_push_notification to either avoid referencing a default implementation via `platforms: android: default_package: stream_video_push_notification` or add an inline implementation to stream_video_push_notification via `platforms: android:` `pluginClass` or `dartPluginClass`.

Package stream_video_push_notification:android references stream_video_push_notification:android as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_video_push_notification to either avoid referencing a default implementation via `platforms: android: default_package: stream_video_push_notification` or add an inline implementation to stream_video_push_notification via `platforms: android:` `pluginClass` or `dartPluginClass`.

Package stream_video_push_notification:android references stream_video_push_notification:android as the default plugin, but it does not provide an inline implementation.
Ask the maintainers of stream_video_push_notification to either avoid referencing a default implementation via `platforms: android: default_package: stream_video_push_notification` or add an inline implementation to stream_video_push_notification via `platforms: android:` `pluginClass` or `dartPluginClass`.
```

**Comment by Mar1155** [comment]
Could you please provide the most recent compatible Flutter versions?

**Comment by rlee1990** [comment]
@Mar1155 I am using flutter version 3.27.1 on stable.
The app will still load but gives the above errors.

**Comment by Mar1155** [comment]
i'm using 3.27.1 on stable too, but the build crashes with this error.

`Launching lib/main.dart on sdk gphone64 arm64 in debug mode...
You are applying Flutter's app_plugin_loader Gradle plugin imperatively using the apply script method, which is deprecated and will be removed in a future release. Migrate to applying Gradle plugins with the declarative plugins block: https://flutter.dev/to/flutter-gradle-plugin-apply

You are applying Flutter's main Gradle plugin imperatively using the apply script method, which is deprecated and will be removed in a future release. Migrate to applying Gradle plugins with the declarative plugins block: https://flutter.dev/to/flutter-gradle-plugin-apply

../packages/stream_video_flutter/lib/src/call_screen/lobby_video.dart:5:8: Error: Error when reading '../packages/stream_video_flutter/lib/stream_video/lib/src/stream_video.dart': No such file or directory
import '../../../../stream_video/lib/src/stream_video.dart';
       ^
../packages/stream_video_flutter/lib/src/call_screen/lobby_view.dart:6:8: Error: Error when reading '../packages/stream_video_flutter/lib/stream_video/lib/src/stream_video.dart': No such file or directory
import '../../../../stream_video/lib/src/stream_video.dart';
       ^
../packages/stream_video_flutter/lib/src/call_screen/lobby_video.dart:6:8: Error: Error when reading '../packages/stream_video_flutter/lib/stream_video/lib/stream_video.dart': No such file or directory
import '../../../../stream_video/lib/stream_video.dart';
       ^
../packages/stream_video_flutter/lib/src/call_screen/lobby_view.dart:7:8: Error: Error when reading '../packages/stream_video_flutter/lib/stream_video/lib/stream_video.dart': No such file or directory
import '../../../../stream_video/lib/stream_video.dart';
       ^
../packages/stream_video_flutter/lib/src/call_participants/call_participants.dart:163:35: Error: The argument type 'CallParticipantState?' can't be assigned to the parameter type 'CallParticipantState' because 'CallParticipantState?' is nullable and 'CallParticipantState' isn't.
 - 'CallParticipantState' is from 'package:stream_video/src/models/call_participant_state.dart' ('../packages/stream_video/lib/src/models/call_participant_state.dart').
        screenSharingParticipant: _screenShareParticipant,
                                  ^
../packages/stream_video_flutter/lib/src/call_screen/lobby_video.dart:144:39: Error: The argument type 'RtcLocalTrack<CameraConstraints>?' can't be assigned to the parameter type 'RtcTrack' because 'RtcLocalTrack<CameraConstraints>?' is nullable and 'RtcTrack' isn't.
 - 'RtcLocalTrack' is from 'package:stream_video/src/webrtc/rtc_track/rtc_local_track.dart' ('../packages/stream_video/lib/src/webrtc/rtc_track/rtc_local_track.dart').
 - 'CameraConstraints' is from 'package:stream_video/src/webrtc/media/camera_constraints.dart' ('../packages/stream_video/lib/src/webrtc/media/camera_constraints.dart').
 - 'RtcTrack' is from 'package:stream_video/src/webrtc/rtc_track/rtc_track.dart' ('../packages/stream_video/lib/src/webrtc/rtc_track/rtc_track.dart').
                          videoTrack: _cameraTrack,
                                      ^
../packages/stream_video_flutter/lib/src/call_screen/lobby_video.dart:50:24: Context: '_cameraTrack' refers to a non-final field so it couldn't be promoted.
See http://dart.dev/go/non-promo-non-final-field
  RtcLocalCameraTrack? _cameraTrack;
                       ^
../packages/stream_video_flutter/lib/src/widgets/stream_user_avatar.dart:155:16: Error: A value of type 'ClipRRect' can't be assigned to a variable of type 'FittedBox'.
 - 'ClipRRect' is from 'package:flutter/src/widgets/basic.dart' ('/Users/mariobarbieri/Developer/flutter/packages/flutter/lib/src/widgets/basic.dart').
 - 'FittedBox' is from 'package:flutter/src/widgets/basic.dart' ('/Users/mariobarbieri/Developer/flutter/packages/flutter/lib/src/widgets/basic.dart').
      avatar = ClipRRect(
               ^
../packages/stream_video_flutter/lib/src/theme/call_content_theme.dart:37:41: Error: The argument type 'Color?' can't be assigned to the parameter type 'Color' because 'Color?' is nullable and 'Color' isn't.
 - 'Color' is from 'dart:ui'.
      callContentBackgroundColor: Color.lerp(
                                        ^
../packages/stream_video_flutter/lib/src/theme/call_controls_theme.dart:137:34: Error: The argument type 'BorderRadius?' can't be assigned to the parameter type 'BorderRadius' because 'BorderRadius?' is nullable and 'BorderRadius' isn't.
 - 'BorderRadius' is from 'package:flutter/src/painting/border_radius.dart' ('/Users/mariobarbieri/Developer/flutter/packages/flutter/lib/src/painting/border_radius.dart').
      borderRadius: BorderRadius.lerp(borderRadius, other.borderRadius, t),
                                 ^
../packages/stream_video_flutter/lib/src/theme/call_controls_theme.dart:138:30: Error: The argument type 'Color?' can't be assigned to the parameter type 'Color' because 'Color?' is nullable and 'Color' isn't.
 - 'Color' is from 'dart:ui'.
      backgroundColor: Color.lerp(backgroundColor, other.backgroundColor, t),
                             ^
../packages/stream_video_flutter/lib/src/theme/call_controls_theme.dart:140:27: Error: The argument type 'EdgeInsets?' can't be assigned to the parameter type 'EdgeInsets' because 'EdgeInsets?' is nullable and 'EdgeInsets' isn't.
 - 'EdgeInsets' is from 'package:flutter/src/painting/edge_insets.dart' ('/Users/mariobarbieri/Developer/flutter/packages/flutter/lib/src/painting/edge_insets.dart').
      padding: EdgeInsets.lerp(padding, other.padding, t),
                          ^
../packages/stream_video_flutter/lib/src/theme/call_controls_theme.dart:142:30: Error: The argument type 'Color?' can't be assigned to the parameter type 'Color' because 'Color?' is nullable and 'Color' isn't.
 - 'Color' is from 'dart:ui'.
      optionIconColor: Color.lerp(optionIconColor, other.optionIconColor, t),
                             ^
../packages/stream_video_flutter/lib/src/theme/call_controls_theme.dart:143:38: Error: The argument type 'Color?' can't be assigned to the parameter type 'Color' because 'Color?' is nullable and 'Color' isn't.
 - 'Color' is from 'dart:ui'.
      inactiveOptionIconColor: Color.lerp(
                                     ^
../packages/stream_video_flutter/lib/src/theme/call_controls_theme.dart:155:17: Error: The argument type 'Color?' can't be assigned to the parameter type 'Color' because 'Color?' is nullable and 'Color' isn't.
 - 'Color' is from 'dart:ui'.
          Color.lerp(optionBackgroundColor, other.optionBackgroundColor, t),
                ^
../packages/stream_video_flutter/lib/src/theme/call_controls_theme.dart:156:44: Error: The argument type 'Color?' can't be assigned to the parameter type 'Color' because 'Color?' is nullable and 'Color' isn't.
 - 'Color' is from 'dart:ui'.
      inactiveOptionBackgroundColor: Color.lerp(
                                           ^
../packages/stream_video_flutter/lib/src/theme/call_controls_theme.dart:161:35: Error: The argument type 'OutlinedBorder?' can't be assigned to the parameter type 'OutlinedBorder' because 'OutlinedBorder?' is nullable and 'OutlinedBorder' isn't.
 - 'OutlinedBorder' is from 'package:flutter/src/painting/borders.dart' ('/Users/mariobarbieri/Developer/flutter/packages/flutter/lib/src/painting/borders.dart').
      optionShape: OutlinedBorder.lerp(optionShape, other.optionShape, t),
                                  ^
../packages/stream_video_flutter/lib/src/theme/call_controls_theme.dart:163:30: Error: The argument type 'EdgeInsetsGeometry?' can't be assigned to the parameter type 'EdgeInsetsGeometry' because 'EdgeInsetsGeometry?' is nullable and 'EdgeInsetsGeometry' isn't.
 - 'EdgeInsetsGeometry' is from 'package:flutter/src/painting/edge_insets.dart' ('/Users/mariobarbieri/Developer/flutter/packages/flutter/lib/src/painting/edge_insets.dart').
          EdgeInsetsGeometry.lerp(optionPadding, other.optionPadding, t),
                             ^
../packages/stream_video_flutter/lib/src/theme/call_participant_theme.dart:180:30: Error: The argument type 'Color?' can't be assigned to the parameter type 'Color' because 'Color?' is nullable and 'Color' isn't.
 - 'Color' is from 'dart:ui'.
      backgroundColor: Color.lerp(backgroundColor, other.backgroundColor, t),
                             ^
../packages/stream_video_flutter/lib/src/theme/call_participant_theme.dart:181:34: Error: The argument type 'BorderRadius?' can't be assigned to the parameter type 'BorderRadius' because 'BorderRadius?' is nullable and 'BorderRadius' isn't.
 - 'BorderRadius' is from 'package:flutter/src/painting/border_radius.dart' ('/Users/mariobarbieri/Developer/flutter/packages/flutter/lib/src/painting/border_radius.dart').
      borderRadius: BorderRadius.lerp(borderRadius, other.borderRadius, t),
                                 ^
../packages/stream_video_flutter/lib/src/theme/call_participant_theme.dart:189:33: Error: The argument type 'Color?' can't be assigned to the parameter type 'Color' because 'Color?' is nullable and 'Color' isn't.
 - 'Color' is from 'dart:ui'.
      speakerBorderColor: Color.lerp(
                                ^
../packages/stream_video_flutter/lib/src/theme/call_participant_theme.dart:196:44: Error: The argument type 'TextStyle?' can't be assigned to the parameter type 'TextStyle' because 'TextStyle?' is nullable and 'TextStyle' isn't.
 - 'TextStyle' is from 'package:flutter/src/painting/text_style.dart' ('/Users/mariobarbieri/Developer/flutter/packages/flutter/lib/src/painting/text_style.dart').
      participantLabelTextStyle: TextStyle.lerp(
                                           ^
../packages/stream_video_flutter/lib/src/theme/call_participant_theme.dart:201:52: Error: The argument type 'AlignmentGeometry?' can't be assigned to the parameter type 'AlignmentGeometry' because 'AlignmentGeometry?' is nullable and 'AlignmentGeometry' isn't.
 - 'AlignmentGeometry' is from 'package:flutter/src/painting/alignment.dart' ('/Users/mariobarbieri/Developer/flutter/packages/flutter/lib/src/painting/alignment.dart').
      participantLabelAlignment: AlignmentGeometry.lerp(
                                                   ^
../packages/stream_video_flutter/lib/src/theme/call_participant_theme.dart:206:39: Error: The argument type 'Color?' can't be assigned to the parameter type 'Color' because 'Color?' is nullable and 'Color' isn't.
 - 'Color' is from 'dart:ui'.
      audioLevelIndicatorColor: Color.lerp(
                                      ^
../packages/stream_video_flutter/lib/src/theme/call_participant_theme.dart:212:17: Error: The argument type 'Color?' can't be assigned to the parameter type 'Color' because 'Color?' is nullable and 'Color' isn't.
 - 'Color' is from 'dart:ui'.
          Color.lerp(enabledMicrophoneColor, other.enabledMicrophoneColor, t),
                ^
../packages/stream_video_flutter/lib/src/theme/call_participant_theme.dart:213:38: Error: The argument type 'Color?' can't be assigned to the parameter type 'Color' because 'Color?' is nullable and 'Color' isn't.
 - 'Color' is from 'dart:ui'.
      disabledMicrophoneColor: Color.lerp(
                                     ^
../packages/stream_video_flutter/lib/src/theme/call_participant_theme.dart:221:41: Error: The argument type 'Color?' can't be assigned to the parameter type 'Color' because 'Color?' is nullable and 'Color' isn't.
 - 'Color' is from 'dart:ui'.
      connectionLevelActiveColor: Color.lerp(
                                        ^
../packages/stream_video_flutter/lib/src/theme/call_participant_theme.dart:226:43: Error: The argument type 'Color?' can't be assigned to the parameter type 'Color' because 'Color?' is nullable and 'Color' isn't.
 - 'Color' is from 'dart:ui'.
      connectionLevelInactiveColor: Color.lerp(
                                          ^
../packages/stream_video_flutter/lib/src/theme/call_participant_theme.dart:231:51: Error: The argument type 'AlignmentGeometry?' can't be assigned to the parameter type 'AlignmentGeometry' because 'AlignmentGeometry?' is nullable and 'AlignmentGeometry' isn't.
 - 'AlignmentGeometry' is from 'package:flutter/src/painting/alignment.dart' ('/Users/mariobarbieri/Developer/flutter/packages/flutter/lib/src/painting/alignment.dart').
      connectionLevelAlignment: AlignmentGeometry.lerp(
                                                  ^
../packages/stream_video_flutter/lib/src/theme/call_participant_theme.dart:236:43: Error: The argument type 'EdgeInsets?' can't be assigned to the parameter type 'EdgeInsets' because 'EdgeInsets?' is nullable and 'EdgeInsets' isn't.
 - 'EdgeInsets' is from 'package:flutter/src/painting/edge_insets.dart' ('/Users/mariobarbieri/Developer/flutter/packages/flutter/lib/src/painting/edge_insets.dart').
      participantsGridPadding: EdgeInsets.lerp(
                                          ^
../packages/stream_video_flutter/lib/src/theme/incoming_outgoing_call_theme.dart:111:45: Error: The argument type 'TextStyle?' can't be assigned to the parameter type 'TextStyle' because 'TextStyle?' is nullable and 'TextStyle' isn't.
 - 'TextStyle' is from 'package:flutter/src/painting/text_style.dart' ('/Users/mariobarbieri/Developer/flutter/packages/flutter/lib/src/painting/text_style.dart').
      singleParticipantTextStyle: TextStyle.lerp(
                                            ^
../packages/stream_video_flutter/lib/src/theme/incoming_outgoing_call_theme.dart:116:47: Error: The argument type 'TextStyle?' can't be assigned to the parameter type 'TextStyle' because 'TextStyle?' is nullable and 'TextStyle' isn't.
 - 'TextStyle' is from 'package:flutter/src/painting/text_style.dart' ('/Users/mariobarbieri/Developer/flutter/packages/flutter/lib/src/painting/text_style.dart').
      multipleParticipantTextStyle: TextStyle.lerp(
                                              ^
../packages/stream_video_flutter/lib/src/theme/incoming_outgoing_call_theme.dart:121:40: Error: The argument type 'TextStyle?' can't be assigned to the parameter type 'TextStyle' because 'TextStyle?' is nullable and 'TextStyle' isn't.
 - 'TextStyle' is from 'package:flutter/src/painting/text_style.dart' ('/Users/mariobarbieri/Developer/flutter/packages/flutter/lib/src/painting/text_style.dart').
      callingLabelTextStyle: TextStyle.lerp(
                                       ^
../packages/stream_video_flutter/lib/src/theme/lobby_view_theme.dart:104:30: Error: The argument type 'Color?' can't be assigned to the parameter type 'Color' because 'Color?' is nullable and 'Color' isn't.
 - 'Color' is from 'dart:ui'.
      backgroundColor: Color.lerp(backgroundColor, other.backgroundColor, t),
                             ^
../packages/stream_video_flutter/lib/src/theme/lobby_view_theme.dart:106:17: Error: The argument type 'Color?' can't be assigned to the parameter type 'Color' because 'Color?' is nullable and 'Color' isn't.
 - 'Color' is from 'dart:ui'.
          Color.lerp(cardBackgroundColor, other.cardBackgroundColor, t),
                ^
../packages/stream_video_flutter/lib/src/theme/lobby_view_theme.dart:115:39: Error: The argument type 'Color?' can't be assigned to the parameter type 'Color' because 'Color?' is nullable and 'Color' isn't.
 - 'Color' is from 'dart:ui'.
      optionOffBackgroundColor: Color.lerp(
                                      ^
../packages/stream_video_flutter/lib/src/theme/lobby_view_theme.dart:121:17: Error: The argument type 'Color?' can't be assigned to the parameter type 'Color' because 'Color?' is nullable and 'Color' isn't.
 - 'Color' is from 'dart:ui'.
          Color.lerp(optionOffIconColor, other.optionOffIconColor, t),
                ^
../packages/stream_video_flutter/lib/src/theme/local_video_theme.dart:104:34: Error: The argument type 'BorderRadius?' can't be assigned to the parameter type 'BorderRadius' because 'BorderRadius?' is nullable and 'BorderRadius' isn't.
 - 'BorderRadius' is from 'package:flutter/src/painting/border_radius.dart' ('/Users/mariobarbieri/Developer/flutter/packages/flutter/lib/src/painting/border_radius.dart').
      borderRadius: BorderRadius.lerp(borderRadius, other.borderRadius, t),
                                 ^
../packages/stream_video_flutter/lib/src/theme/local_video_theme.dart:105:26: Error: The argument type 'Color?' can't be assigned to the parameter type 'Color' because 'Color?' is nullable and 'Color' isn't.
 - 'Color' is from 'dart:ui'.
      shadowColor: Color.lerp(shadowColor, other.shadowColor, t),
                         ^
../packages/stream_video_flutter/lib/src/theme/stream_livestream_theme.dart:138:30: Error: The argument type 'Color?' can't be assigned to the parameter type 'Color' because 'Color?' is nullable and 'Color' isn't.
 - 'Color' is from 'dart:ui'.
      liveButtonColor: Color.lerp(liveButtonColor, other.liveButtonColor, t),
                             ^
../packages/stream_video_flutter/lib/src/theme/stream_livestream_theme.dart:140:17: Error: The argument type 'Color?' can't be assigned to the parameter type 'Color' because 'Color?' is nullable and 'Color' isn't.
 - 'Color' is from 'dart:ui'.
          Color.lerp(backstageButtonColor, other.backstageButtonColor, t),
                ^
../packages/stream_video_flutter/lib/src/theme/stream_livestream_theme.dart:141:43: Error: The argument type 'TextStyle?' can't be assigned to the parameter type 'TextStyle' because 'TextStyle?' is nullable and 'TextStyle' isn't.
 - 'TextStyle' is from 'package:flutter/src/painting/text_style.dart' ('/Users/mariobarbieri/Developer/flutter/packages/flutter/lib/src/painting/text_style.dart').
      callStateButtonTextStyle: TextStyle.lerp(
                                          ^
../packages/stream_video_flutter/lib/src/theme/stream_livestream_theme.dart:146:44: Error: The argument type 'TextStyle?' can't be assigned to the parameter type 'TextStyle' because 'TextStyle?' is nullable and 'TextStyle' isn't.
 - 'TextStyle' is from 'package:flutter/src/painting/text_style.dart' ('/Users/mariobarbieri/Developer/flutter/packages/flutter/lib/src/painting/text_style.dart').
      participantCountTextStyle: TextStyle.lerp(
                                           ^
../packages/stream_video_flutter/lib/src/theme/stream_livestream_theme.dart:151:36: Error: The argument type 'TextStyle?' can't be assigned to the parameter type 'TextStyle' because 'TextStyle?' is nullable and 'TextStyle' isn't.
 - 'TextStyle' is from 'package:flutter/src/painting/text_style.dart' ('/Users/mariobarbieri/Developer/flutter/packages/flutter/lib/src/painting/text_style.dart').
      durationTextStyle: TextStyle.lerp(
                                   ^
../packages/stream_video_flutter/lib/src/theme/user_avatar_theme.dart:71:34: Error: The argument type 'BorderRadius?' can't be assigned to the parameter type 'BorderRadius' because 'BorderRadius?' is nullable and 'BorderRadius' isn't.
 - 'BorderRadius' is from 'package:flutter/src/painting/border_radius.dart' ('/Users/mariobarbieri/Developer/flutter/packages/flutter/lib/src/painting/border_radius.dart').
      borderRadius: BorderRadius.lerp(borderRadius, other.borderRadius, t),
                                 ^
../packages/stream_video_flutter/lib/src/theme/user_avatar_theme.dart:72:35: Error: The argument type 'BoxConstraints?' can't be assigned to the parameter type 'BoxConstraints' because 'BoxConstraints?' is nullable and 'BoxConstraints' isn't.
 - 'BoxConstraints' is from 'package:flutter/src/rendering/box.dart' ('/Users/mariobarbieri/Developer/flutter/packages/flutter/lib/src/rendering/box.dart').
      constraints: BoxConstraints.lerp(constraints, other.constraints, t),
                                  ^
../packages/stream_video_flutter/lib/src/theme/user_avatar_theme.dart:74:21: Error: The argument type 'TextStyle?' can't be assigned to the parameter type 'TextStyle' because 'TextStyle?' is nullable and 'TextStyle' isn't.
 - 'TextStyle' is from 'package:flutter/src/painting/text_style.dart' ('/Users/mariobarbieri/Developer/flutter/packages/flutter/lib/src/painting/text_style.dart').
          TextStyle.lerp(initialsTextStyle, other.initialsTextStyle, t),
                    ^
../packages/stream_video_flutter/lib/src/theme/user_avatar_theme.dart:77:29: Error: The argument type 'Color?' can't be assigned to the parameter type 'Color' because 'Color?' is nullable and 'Color' isn't.
 - 'Color' is from 'dart:ui'.
      selectionColor: Color.lerp(selectionColor, other.selectionColor, t),
                            ^
../packages/stream_video_flutter/lib/src/call_participants/layout/call_participants_grid_view.dart:264:13: Error: A value of type 'Widget?' can't be assigned to a variable of type 'Widget' because 'Widget?' is nullable and 'Widget' isn't.
 - 'Widget' is from 'package:flutter/src/widgets/framework.dart' ('/Users/mariobarbieri/Developer/flutter/packages/flutter/lib/src/widgets/framework.dart').
            child,
            ^
../packages/stream_video_flutter/lib/src/widgets/floating_view/snapping_floating_view.dart:140:35: Error: A value of type 'Offset?' can't be assigned to a variable of type 'Offset' because 'Offset?' is nullable and 'Offset' isn't.
 - 'Offset' is from 'dart:ui'.
        _moveOffset = _snapOffsets[_currentAlignment];
                                  ^
../packages/stream_video/lib/src/utils/cancelable_operation.dart:7:67: Error: A value of type 'T?' can't be returned from a function with return type 'FutureOr<T>' because 'T?' is nullable and 'T' isn't.
    return valueOrCancellation(cancellationValue).then((value) => value);
                                                                  ^
../packages/stream_video/lib/src/webrtc/rtc_manager.dart:382:29: Error: Property 'receiver' cannot be accessed on 'RTCRtpTransceiver?' because it is potentially null.
 - 'RTCRtpTransceiver' is from 'package:webrtc_interface/src/rtc_rtp_transceiver.dart' ('/Users/mariobarbieri/.pub-cache/hosted/pub.dev/webrtc_interface-1.2.0/lib/src/rtc_rtp_transceiver.dart').
Try accessing using ?. instead.
      receiver: transceiver.receiver,
                            ^^^^^^^^
../packages/stream_video/lib/src/webrtc/rtc_manager.dart:455:29: Error: Property 'receiver' cannot be accessed on 'RTCRtpTransceiver?' because it is potentially null.
 - 'RTCRtpTransceiver' is from 'package:webrtc_interface/src/rtc_rtp_transceiver.dart' ('/Users/mariobarbieri/.pub-cache/hosted/pub.dev/webrtc_interface-1.2.0/lib/src/rtc_rtp_transceiver.dart').
Try accessing using ?. instead.
      receiver: transceiver.receiver,
                            ^^^^^^^^
Target kernel_snapshot_program failed: Exception


FAILURE: Build failed with an exception.

* Where:
Script '/Users/mariobarbieri/Developer/flutter/packages/flutter_tools/gradle/src/main/groovy/flutter.groovy' line: 1685

* What went wrong:
Execution failed for task ':app:compileFlutterBuildDebug'.
> Process 'command '/Users/mariobarbieri/Developer/flutter/bin/flutter'' finished with non-zero exit value 1

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 14s
Running Gradle task 'assembleDebug'...                             15.5s
Error: Gradle task assembleDebug failed with exit code 1`

**Comment by Brazol** [comment]
@rlee1990 The issue should now be resolved with the latest release (v0.6.1). Thank you for bringing it to our attention.
@Mar1155 I'm closing this issue since it has been resolved. If you still encounter problems with the latest release, please create a separate issue. Thank you!
